src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -281,32 +285,60 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                     }
                 }
             }
-        } else if (args.length >= 1 && args[0].equalsIgnoreCase("groups")) {
-            if (this.getPermissions(channel).getListGroups().isEmpty()) {
-                channel.sendMessage(MessageUtils.getEmbed(sender)
-                        .setColor(Color.BLUE)
-                        .setDescription("There are no groups for this guild!")
-                        .build()).queue();
-                return;
-            } else {
-                int page = args.length == 2 ? Integer.valueOf(args[4]) : 1;
-                Set<String> groups = this.getPermissions(channel).getGroups().keySet();
-                List<String> groupList = GeneralUtils.orderList(groups);
+        } else if (args.length >= 1) {
+            if (args[0].equalsIgnoreCase("groups")) {
+                if (this.getPermissions(channel).getListGroups().isEmpty()) {
+                    channel.sendMessage(MessageUtils.getEmbed(sender)
+                            .setColor(Color.BLUE)
```

Comment:
Why is this not red?

Reply:
Because it really isn't an error. I'll change it anyways